recipes/parallelio/conda_build_config.yaml

```diff
@@ -0,0 +1,11 @@
+mpi:
+  - openmpi
+  - mpich
+
+# temporary because mambabuild seems to be getting confused
+hdf5:
+  - 1.12.1
+libnetcdf:
+  - 4.8.1
+netcdf_fortran:
+  - 4.5
```

Comment:
As commented, here, I wasn't able to get `conda mambabuild` to run locally without these pins, taken from `conda-forge-pinning`.  I kept seeing errors about not finding:
```
netcdf-fortran mpi_openmpi.*
```
I could not figure out why it was getting confused about a constraint on a build string that it seems to think is on the version.

Reply:
I will remove these as soon as this is in its own feedstock.

Reply:
```
netcdf-fortran mpi_openmpi.*
```
Above is an invalid package spec. It should be something like this:

```
netcdf-fortran * mpi_openmpi_*
```
or
```
netcdf-fortran =*=mpi_openmpi_*
```

Reply:
I assume this problem is because you weren't using the `build-locally.py` script? This script does environment/recipe augmentation stuff in addition to starting a docker container.

Reply:
I can't install docker on my laptop because it interferes with my ability to use wifi on my laptop at work.  I can take these out and see if CI does okay.

Reply:
I see you went ahead and took these out, and things built.  Thanks.  Thanks for the tip about `build-locally.py`. I'll see if I can find a solution where I can install Docker and still use LANL's wifi.

Reply:
Pretty it works with podman too in case that helps.

Reply:
That could be worth a try, thanks!